source/docs/zero-to-robot/step-1/how-to-wire-a-robot.rst

```diff
@@ -33,34 +33,34 @@ Locate the following control system components and tools
       - 2x PWM y-cables
       - 120A Circuit breaker
       - 4x 40A Circuit breaker
-      - 6 AWG Red wire
-      - 10 AWG Red/Black wire
-      - 18 AWG Red/Black wire
-      - 22AWG yellow/green twisted CAN cable
-      - 16x 10-12 AWG  (yellow) ring terminals
+      - 6 AWG (16 :math:`mm^2`) Red wire
+      - 10 AWG (6 :math:`mm^2`) Red/Black wire
+      - 18 AWG (1 :math:`mm^2`) Red/Black wire
+      - 22AWG (0.5 :math:`mm^2`) Yellow/Green twisted CAN cable
+      - 16x 10-12 AWG (4 - 6 :math:`mm^2`) (Yellow) ring terminals
         (8x quick disconnect pairs if using integrated wire controllers)
       - 2x Andersen SB50 battery connectors
```

Comment:
`Andersen->Anderson`

Reply:
Found a second one of these.  Good catch